include/homestead_pd_definitions.h

```diff
@@ -74,14 +74,13 @@ static const PDLog1<int> CL_HOMESTEAD_CASSANDRA_CACHE_INIT_FAIL
 (
   PDLogBase::CL_HOMESTEAD_ID + 6,
   LOG_ERR,
-  "Fatal - Failed to initialize the cache for the CassandraStore - "
+  "Fatal - Failed to initialize the Cassandra cache - "
```

Comment:
I've tweaked it to refer to the cassandra "connection", which is a more accurate description of what the `Cache` in homestead is.